tests/test_b4uleave.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict
+from platzky.platzky import create_app_from_config, Config
+
+
+def test_that_plugin_loads_b4uleave():
+
+    data_with_plugin: Dict[str, Any] = {
+        "APP_NAME": "testingApp",
+        "SECRET_KEY": "secret",
+        "USE_WWW": False,
+        "BLOG_PREFIX": "/",
+        "TRANSLATION_DIRECTORIES": ["/some/fake/dir"],
+        "DB": {
+            "TYPE": "json",
+            "DATA": {
+                "site_content": {"pages": []},
+                "plugins": [
+                    {
+                        "name": "b4uleave",
+                        "config": {
+                        },
+                    }
+                ],
+            },
+        },
+    }
+
+    b4uleave_function = "B4ULeave-OknoModalne"
+
+    config_with_plugin = Config.model_validate(data_with_plugin)
+
+    app_with_plugin = create_app_from_config(config_with_plugin)
+
+    response = app_with_plugin.test_client().get("/")
+    
+    assert response.status_code == 404
```

Comment:
this should be 200.

Reply:
custom message configured in config (now 20 line) should be checked in this test.